src/ZEO/tests/test_client_side_conflict_resolution.py

```diff
@@ -1,10 +1,13 @@
+import os
+import shutil
+import tempfile
 import unittest
-import zope.testing.setupstack
 
+import zope.testing.setupstack
```

Comment:
I don't see this new import being used?

Reply:
I moved it around, it's not actually new.

This module defines a base class for the test case classes with some helpers (that became  obsolete when stdlib's unittest gained `self.addCleanup()` back in 2.7).

Reply:
D'oh. I don't know how I missed the `-` line. Sorry!